### PR TITLE
Adds volatility to magnesium-ammonium chloride and fixes a single line of text

### PR DESCRIPTION
**Pull request by SpacemanHenry**
[BUG - TRIVIAL]
[FEATURE]

## About the PR 
Adds a volatility stat of 2 to magnesium-ammonium chloride, allowing it to be used in pipe bombs to decent effect. Since magnesium-ammonium chloride releases so much ammonia when it decomposes, it sounds good for jamming into a closed vessel.

Also fixes a single line of text for fireworks. Specifically, bootleg fireworks being described as empty during the cutting and emptying process, which doesn't make a lick of sense.

## Why's this needed? 
IIRC the only pipe bomb reagent currently available without access to a chem dispenser is welding fuel which isn't very powerful at all. So, adding volatility to magnesium-ammonium chloride would complete a more powerful yet time-consuming option for any ne'er-do-wells without access to chem equipment.

### CODE REVIEW

**Comment by ZeWaka** [comment]
please label your PRs

File: code/obj/item/grenades.dm [hunk 1]
```diff
@@ -979,10 +979,10 @@ PIPE BOMBS + CONSTRUCTION
 
 	attackby(obj/A as obj, mob/user as mob) // adapted from iv_drips.dm
 		if (iscuttingtool(A) && !(src.slashed) && !(src.primed))
+			boutput(user, "You carefully cut [src] open and dump out the contents.")
 			src.slashed = TRUE
 			src.name = "empty [src.name]" // its empty now!
 			src.desc = "[src.desc] It has been cut open and emptied out."
-			boutput(user, "You carefully cut [src] open and dump out the contents.")
```
Comment (ZeWaka) [inline code/obj/item/grenades.dm:985]: don't see how this changes/fixes anything

Comment (Tarmunora) [reply code/obj/item/grenades.dm:985]: moving the print before the name change I assume

Comment (SpacemanHenry) [reply code/obj/item/grenades.dm:985]: Exactly

**Comment by SpacemanHenry** [comment]
> please label your PRs

Will do as soon as I'm not mobile

File: code/obj/item/grenades.dm (hunk 1 above)
Comment (ZeWaka) [reply code/obj/item/grenades.dm:985]: ahh!

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**